content/en/apps/guides/hosting/4.x/upgrading.md

```diff
@@ -0,0 +1,238 @@
+---
+title: "Upgrading from CHT 3.x to CHT 4.x"
+linkTitle: "Upgrading to 4.x"
+weight: 10
+description: >
+  How to upgrade from CHT 3.x to CHT 4.x
+aliases:
+relatedContent: >
+---
+
+The hosting architecture differs entirely between CHT-Core 3.x and CHT-Core 4.x. Thus upgrading involves some manual steps.
+
+### 0. Ensure all clients have successfully synced
+This is not necessary. Should this be a step?
```

Comment:
I'd remove it. Operationally it's very difficult to organise so if it's not necessary we shouldn't suggest it.